proposals/0000-style-script-defaults.md

```diff
@@ -0,0 +1,88 @@
+- Start Date: 2022-03-15
+- Reference Issues: [RFC0008](https://github.com/withastro/rfcs/tree/main/proposals/0008-style-script-behavior.md), [#65](https://github.com/withastro/rfcs/discussions/65)
+- Implementation PR: <!-- leave empty -->
+
+# Summary
+
+Astro is inconsist between `<style>` and `<script>` default behavior. Currently, `<style>` has opt-out processing, but `<script>` does not. This RFC aims to settle on a good, consistent default for both `<style>` and `<script>`.
+
+- New `is:inline` directive to avoid bundling `style` or `script`
+- `<style>` => remains scoped, bundled by default. `is:scoped` directive supported for consistency
```

Comment:
Since scoping is the default, removing the `is:scoped` directive may avoid some confusion

Not a blocker for me at all, there is value in the option for making it explicit so I could go either way with it